src/components/pages/ChatContainer/__tests__/ChatContainer.visual.spec.tsx

```diff
@@ -253,6 +253,34 @@ test.describe('ChatContainer', {tag: '@ChatContainer'}, () => {
         await expectScreenshot();
     });
 
+    test('should hide title on empty chat', async ({mount, page, expectScreenshot}) => {
+        await mount(<ChatContainerStories.HiddenTitleOnEmpty />);
+
+        // Check that title is not visible in empty state
+        await expect(page.getByText('AI Chat Assistant')).not.toBeVisible();
+
+        // Take screenshot of empty state without title
+        await expectScreenshot();
+    });
+
+    test('should show title after sending message', async ({mount, page}) => {
+        await mount(<ChatContainerStories.HiddenTitleOnEmpty />);
+
+        // Initially title should be hidden
+        await expect(page.getByText('AI Chat Assistant')).not.toBeVisible();
+
+        // Send a message
+        const textarea = page.locator('textarea');
+        await textarea.fill('Test message');
+        await textarea.press('Enter');
+
+        // Wait for message to be sent and title to appear
+        await page.waitForTimeout(300);
```

Comment:
this can be removed, `toBeVisible()` playwright method already has a built-in wait for text to appear.